app/models/chat_channel_membership.rb

```diff
@@ -18,7 +20,10 @@ class ChatChannelMembership < ApplicationRecord
     ranking ["desc(channel_last_message_at)"]
   end
 
-  private
+  include Searchable
```

Comment:
I'd put this at the top, it's easier to see what modules a class includes right away